app/modules/collaborations/resources.py

```diff
@@ -47,7 +47,7 @@ class Collaborations(Resource):
         },
     )
     @api.parameters(PaginationParameters())
-    @api.response(schemas.BaseCollaborationSchema(many=True))
+    @api.response(schemas.DetailedCollaborationSchema(many=True))
```

Comment:
This was the one line needed to make 418 the same so it really didn't need a separate PR